client/src/main/scala/drt/client/services/SPACircuit.scala

```diff
@@ -82,7 +82,7 @@ case class RootModel(applicationVersion: Pot[ClientServerVersions] = Empty,
                      displayAlertDialog: Pot[Boolean] = Empty,
                      oohStatus: Pot[OutOfHoursStatus] = Empty,
                      featureFlags: Pot[Map[String, Boolean]] = Empty,
-                     fileUploadState:Pot[FileUploadState] = Empty,
+                     fileUploadState:Pot[FileUploadState] = Pot.fromOption(Some(FileUploadState("","")))
```

Comment:
This is equivalent to writing `Ready(FileUploadState("", ""))`

This is not _really_ a semantic way to use `Pot`s you should rather make this an `Option[Pot[FileUploadState]]` and default it to `None`

Reply:
Or alternatively, you could check if the Pot is `Empty` in your view before rendering the form.

Reply:
we need value as component it not initialised it won't show anything on the page.